src/doc/rustc/src/platform-support.md

```diff
@@ -300,7 +300,7 @@ target | std | host | notes
 [`armv8r-none-eabihf`](platform-support/armv8r-none-eabihf.md) | * |  | Bare Armv8-R, hardfloat
 [`armv7a-nuttx-eabi`](platform-support/nuttx.md) | ✓ |  | ARMv7-A with NuttX
 [`armv7a-nuttx-eabihf`](platform-support/nuttx.md) | ✓ |  | ARMv7-A with NuttX, hardfloat
-`avr-unknown-gnu-atmega328` | * |  | AVR. Requires `-Z build-std=core`
+`avr-none` | * |  | AVR. Requires `-Z build-std=core`
```

Comment:
Could you add a note about target-cpu here as well? E.g. https://github.com/rust-lang/rust/blob/56795fb77a69bc61836537175564455067adada6/src/doc/rustc/src/platform-support.md?plain=1#L275

Reply:
Okie, added.